tests/roots/test-ext-autodoc/target/genericalias.py

```diff
@@ -3,14 +3,21 @@
 from typing import Callable, List
 
 #: A list of int
-T = List[int]
+T1 = List[int]
```

Comment:
Try with docstrings of the form

```
T3 = list[int]
"""Some docstring afterwards?"
``` 

I don't remember whether we picked up those docstrings as docstrings or not by the way so maybe this test is not needed.

Reply:
Addressed in 9ab6198d502df3ae31820e8412d498ebb6c32290